auth/src/main/java/org/openedx/auth/presentation/restore/RestorePasswordFragment.kt

```diff
@@ -188,6 +197,7 @@ private fun RestorePasswordScreen(
 
             Surface(
                 modifier = Modifier
+                    .testTag("txt_forgot_password_description")
```

Comment:
I don't think we need one here. Plus the id is a duplicate of the actual description.

Reply:
good catch.